sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/DateTimeUtils.scala

```diff
@@ -237,7 +237,13 @@ object DateTimeUtils {
    * Converts milliseconds since the epoch to microseconds.
    */
   def millisToMicros(millis: Long): Long = {
-    Math.multiplyExact(millis, MICROS_PER_MILLIS)
+    if (millis < MIN_MICROS) {
+      Long.MinValue
```

Comment:
This is the wrong answer though. Where does overflow arise? if a user asks to convert such a large value, I think the error is not unreasonable